engine/apps/labels/tests/test_labels_cache.py

```diff
@@ -39,6 +40,14 @@ def test_update_labels_cache_for_key(make_organization, make_label_key_and_value
     assert label_value2.name == old_value2_name
 
 
+@pytest.mark.django_db
+def test_update_labels_cache_error(make_organization, make_label_key_and_value, make_label_value):
```

Comment:
This test should be also removed, once we will process all tasks with wrong args. right? It's not an expected case, when we do not catch exception.

Reply:
makes sense :+1: